### ISSUE TITLE
Merge with main project

### ISSUE THREAD

**Issue by velo** [OpenFeign/feign-form#127]
I tried to reach maintainers over email, but no joy.

I would like to merge this 2 modules into the main project.

https://github.com/OpenFeign/feign/pull/2559

All history would be kept.

Also, please let me know if you would like to get committer access to new repo

@xxlabaza @tjuchniewicz

**Comment by xxlabaza** [comment]
Hi!

I'm the author of this module. Sorry, I didn't see your email. 

I tried to come back to the project recently, update it, and release a new version, but it looks like my CI/CD pipeline is broken, so I couldn't make a new release, and because of lack of time, I postponed that.

Yep, it would be great to merge the project with the main one because I don't have enough time to maintain it properly. I don't see many options here for improvements (multiple OOM complaints in this repo - Feign's request template specific, there's nothing to do with that), but maybe I'm wrong. 

Anyway, I would be happy if the community could have access to this module. And yes, getting committer access to the new repo would be great.
